app/models/embed/purl.rb

```diff
@@ -183,9 +189,14 @@ def has_image_data?
         def has_location_data?
           @file.xpath('./location[@type="url"]').present?
         end
+        def has_geo_data?
```

Comment:
I think this method is noop too. Either should be moved and tested/used, or removed